primitives/api/proc-macro/src/attribute_names.rs

```diff
@@ -0,0 +1,24 @@
+/// The ident used for the block generic parameter.
```

Comment:
I need `API_VERSION_ATTRIBUTE` in both macros so I extracted these in separate file.